pywinauto/controls/common_controls.py

```diff
@@ -3388,6 +3398,87 @@ def set_view(self, viewType):
         if res == 0:
             raise RuntimeError('Failed to set view in Calendar')
 
+    # ----------------------------------------------------------------
+    def set_id(self, ID):
+        """
+        Set the calendar type.
+
+        Receive only one parameter, which takes variants below:
+        'gregorian', 'gregorian_us', 'japan', 'taiwan', 'korea',
+        'hijri', 'thai', 'hebrew', 'gregorian_me_french',
+        'gregorian_arabic', 'gregorian_english_xlit',
+        'gregorian_french_xlit', 'umalqura'
+        """
+
+        dict_types = {
+            'gregorian': win32defines.CAL_GREGORIAN,
+            'gregorian_us': win32defines.CAL_GREGORIAN_US,
+            'japan': win32defines.CAL_JAPAN,
+            'taiwan': win32defines.CAL_TAIWAN,
+            'korea': win32defines.CAL_KOREA,
+            'hijri': win32defines.CAL_HIJRI,
+            'thai': win32defines.CAL_THAI,
+            'hebrew': win32defines.CAL_HEBREW,
+            'gregorian_me_french': win32defines.CAL_GREGORIAN_ME_FRENCH,
+            'gregorian_arabic': win32defines.CAL_GREGORIAN_ARABIC,
+            'gregorian_english_xlit': win32defines.CAL_GREGORIAN_XLIT_ENGLISH,
+            'gregorian_french_xlit': win32defines.CAL_GREGORIAN_XLIT_FRENCH,
+            'umalqura': win32defines.CAL_UMALQURA
+        }
+        if ID in dict_types:
+            self.send_message(win32defines.MCM_SETCALID, dict_types[ID], 0)
+        else:
+            raise ValueError('Incorrect calendar ID (use one of {0})'.format(dict_types.keys()))
+
+    # ----------------------------------------------------------------
+    def get_id(self):
+        """Get type of calendar"""
+        return self.send_message(win32defines.MCM_GETCALID, 0, 0)
+
+    # ----------------------------------------------------------------
+    def set_color(self, place_of_color, red, green, blue):
+        """
+        Set some color in some place of calendar which you specify.
+
+        Receive four parameters:
+        - The first parameter may take few variants below:
+        'background', 'month_background', 'text', 'title_background',
+        'title_text', 'trailing_text' ;
+        - All other parameters should be integer from 0 to 255.
+        """
+
+        if not (0 <= red <= 255):
+            raise RuntimeError('Incorrect range of color, must be from 0 to 255')
+        if not (0 <= green <= 255):
+            raise RuntimeError('Incorrect range of color, must be from 0 to 255')
+        if not (0 <= blue <= 255):
+            raise RuntimeError('Incorrect range of color, must be from 0 to 255')
+        str_red = str(red)
+        str_green = str(green)
+        str_blue = str(blue)
+        color = str_red + str_green + str_blue

```

Comment:
I read this docs many times but don't understand how I can reach this mask: 0x00bbggrr
I try in Python CMD test byte operations. This operations not using with tuple, only with int.
Every try with ">>16", ">> 8", and another ">>"with int f color give me 0x0 in result.
I try some hex operations, best what I made it:
hex(0) + hex(blye)[2:] + hex(green)[2:] + hex(red)[2:]

But I think it's not what you want.